src/include/migraphx/op/rnn_last_hs_output.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_RNN_LAST_HS_OUTPUT_HPP
+#define MIGRAPHX_GUARD_OPERATORS_RNN_LAST_HS_OUTPUT_HPP
+
+#include <migraphx/op/rnn_var_sl_last_output.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace op {
+
+struct rnn_last_hs_output : rnn_var_sl_last_output
```

Comment:
Why does this inherit `rnn_var_sl_last_output`?

Reply:
I want to reuse the compute_shape() function of rnn_var_sl_last_out, just fixed following your suggestion